oscarapi/serializers/basket.py

```diff
@@ -126,6 +128,18 @@ class Meta:
             'is_tax_known', 'warning', 'basket', 'stockrecord', 'date_created'
         ))
 
+    def update(self, instance, validated_attrs):
+        # This is needed to reflect offer discounts or strategy related
+        # prices immediately in the response
+        super(BasketLineSerializer, self).update(instance, validated_attrs)
+        instance = self.Meta.model.objects.get(id=instance.id)
```

Comment:
maybe this is not needed? Maybe we can clear the basket line cache some other way and then reapply the strategy?